test/view_model_tests/pre_auth_view_models/login_view_model_test.dart

```diff
@@ -31,123 +39,127 @@ final data = {
-  //       data: data,
-  //       options: QueryOptions(document: gql(queries.loginUser('', ''))),
-  //     );
-
-  //     when(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')))
-  //         .thenAnswer((_) async => result);
-
-  //     await model.login();
-
-  //     expect(model.validate, AutovalidateMode.disabled);
-
-  //     verify(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')));
-  //   });
-  //   testWidgets('Check if login() is working fine when invalid credentials',
-  //       (tester) async {
-  //     reset(navigationService);
-  //     final model = LoginViewModel();
-
-  //     await tester.pumpWidget(
-  //       Form(
-  //         key: model.formKey,
-  //         child: Container(),
-  //       ),
-  //     );
-
-  //     when(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')))
-  //         .thenAnswer((_) async => null);
-
-  //     await model.login();
-
-  //     expect(model.validate, AutovalidateMode.disabled);
-
-  //     verify(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')));
-
-  //     verifyNever(
-  //       navigationService.removeAllAndPush(
-  //         Routes.waitingScreen,
-  //         Routes.splashScreen,
-  //       ),
-  //     );
-  //   });
-  //   testWidgets('Check if login() is working fine when throws error',
-  //       (tester) async {
-  //     final model = LoginViewModel();
-
-  //     await tester.pumpWidget(
-  //       Form(
-  //         key: model.formKey,
-  //         child: Container(),
-  //       ),
-  //     );
-
-  //     when(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')))
-  //         .thenThrow(Exception());
-
-  //     await model.login();
-
-  //     expect(model.validate, AutovalidateMode.disabled);
-
-  //     verify(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')));
-  //   });
-  // });
+  // setupFirebaseMocks();
+  // await Firebase.initializeApp();
+  // FirebaseMessagingPlatform.instance = kMockMessagingPlatform;
+
+  setUp(() async {
+    locator.registerSingleton(Queries());
+    registerServices();
+    await locator.unregister<UserConfig>();
+  });
+  tearDown(() async {
+    await locator.unregister<Queries>();
+  });
+
+  group('LoginViewModel Test -', () {
+    testWidgets(
+        'Check if login() is working fine when organisation is not empty',
+        (tester) async {
+      locator.registerSingleton<UserConfig>(MockUserConfig());
+
+      final model = LoginViewModel();
+
+      await tester.pumpWidget(
+        Form(
+          key: model.formKey,
+          child: Container(),
+        ),
+      );
+
+      final result = QueryResult(
+        source: QueryResultSource.network,
+        data: data,
+        options: QueryOptions(document: gql(queries.loginUser('', ''))),
+      );
+
+      when(databaseFunctions.gqlNonAuthMutation(queries.loginUser('', '')))
+          .thenAnswer((_) async => result);
+
+      await model.login();
+
```

Comment:
The newline after await and expect seem extraneous. Please remove them all over the file.